### PR TITLE
navigation component done

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
Navigation component

### CODE REVIEW

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.nav-icon {
+    color: var(--primary-color);
+    font-size: 2rem;
+}
+.nav-component {
+    display: flex;
+    justify-content: space-around;
+    gap: 12px;
```
Comment (pariyar07) [inline Components/Navigation/navigation.css:16]: try using rem here

**Review by pariyar07 — APPROVED** [review]
Love the folder structure. Good to merge.

File: Components/Navigation/navigation.css (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Navigation/navigation.css:16]: sure

**Review by Aditi0505 — APPROVED** [review]
- Good class names
- Good use of Semantic tags
- Easily readable code

Suggestion: Add description and screenshots from next time.

**Pull request merged**